gslib/commands/rm.py

```diff
@@ -66,9 +66,9 @@
   will remove all objects under gs://bucket/subdir or any of its
   subdirectories.
 
-  You can also use the -r option to specify recursive object deletion. Thus, for
-  example, either of the following two commands will remove gs://bucket/subdir
-  and all objects and subdirectories under it:
+  You can also use the -r option to specify recursive object and/or bucket
```

Comment:
I'd recommend not adding "and/or bucket" here, since the accompanying example does not remove the bucket, and because there is discussion of using -r to delete a bucket later on in the page.